Sources/VLCMovieViewController.m

```diff
@@ -344,6 +352,24 @@ - (void)setupControlPanel
     [self.view addSubview:_controllerPanel];
 }
 
+-(NSArray *)getPlaybackControlToolbarConstraints
+{
+    float margin = 40.0f;
+    UILayoutGuide *guide;
+    if(@available(iOS 11.0, *)) {
+        guide = self.view.safeAreaLayoutGuide;
+    } else {
+        guide = self.view.layoutMarginsGuide;
+    }
```

Comment:
same as in another PR set the guide on declaration to self.view.layoutMarginsGuide